starter-code/app.js

```diff
@@ -54,5 +47,13 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+//1 if there is nothing inside index, it will go on into the next one to find whats on celebrities
+const celebritiesRoutes = require('./routes/celebrities.routes')
+app.use('/', celebritiesRoutes)
```

Comment:
a different approach to setting your routes in app.js is to add the first part of the url here. /celebrities since you know all the routes inside will have that. That way only the requests that start with /celebrities will access this file. Same with /movies below.